bool: false
k_pos_z : 2.2361
k_pos_x : 3.1623
k_pos_y : 3.1623
k_vel_z : 3.0777
k_vel_x : 8.4827
k_vel_y : 8.4827
k_acc_z : 0
k_acc_x : 18.7962 
k_acc_y : 18.7962 
k_jer_z : 0
k_jer_x : 17.4399 
k_jer_y : 17.4399 
m_quad : 1.6    
m_load : 0.25   
l_cable : 0.85  
gravity : 9.8  
att_ctrl_tau : 0.3